ocaml/idl/datamodel.ml

```diff
@@ -6577,6 +6579,25 @@ let vdi_disable_cbt = call
     ~allowed_roles:_R_VM_ADMIN
     ()
 
+let vdi_hidden_cbt_setter = call
+    ~name:"hidden_cbt_setter"
+    ~in_oss_since:None
+    ~in_product_since:rel_inverness
+    ~params:[Ref _vdi, "self", "The VDI for which CBT enabled status should be set";
+             Bool, "value", "The value to set"]
+    ~errs:[
+      Api_errors.sr_operation_not_supported;
```

Comment:
We should have an empty error list here as this function doesn't throw any errors.

Reply:
ok, will update this